test/integration/resourcemanager/managedresource/resource_test.go

```diff
@@ -786,10 +786,10 @@ var _ = Describe("ManagedResource controller tests", func() {
 				updatedDeployment.Spec.Template = *newPodTemplateSpec
 				Expect(testClient.Update(ctx, updatedDeployment)).To(Succeed())
 
-				Consistently(func(g Gomega) bool {
+				Eventually(func(g Gomega) corev1.ResourceRequirements {
```

Comment:
Shouldn't we use `Consistently` itself here?

Reply:
Good catch, thank you. Changed it back.